atomic_reactor/plugins/exit_verify_media_types.py

```diff
@@ -98,7 +98,9 @@ def get_manifest_list_only_expectation(self):
 
         :return: bool, expect manifest list only?
         """
-        if not self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY):
+        group_types = [MEDIA_TYPE_DOCKER_V2_MANIFEST_LIST, MEDIA_TYPE_OCI_V1_INDEX]
+        manifest_results = self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY)
+        if not manifest_results or manifest_results.get("media_type") not in group_types:
```

Comment:
Do we want to use the new `is_manifest_list` function here?

Reply:
Yes, fixed.